generated/.tailcallrc.schema.json

```diff
@@ -1474,7 +1474,9 @@
           "enum": [
             "Str",
             "Num",
-            "Bool"
+            "Bool",
+            "Empty",
+            "Any"
```

Comment:
tailcallrc should also have the generated types.

Reply:
you mean `.tailcallrc.graphql`? 
yeah, that types are not populated on generation.
The issue is with graphql syntax at least since JsonSchema type is generated as 

```
input Schema {
  Obj: JSON
  Arr: Schema
  Opt: Schema
  Enum: [String!]
}
```

We can solve it by adding nesting to scalar types in schema instead of storing simple types in root enum so output could look like 
```
```